### DISCUSSION TITLE
Is Nuxt 3 RC13 Broken??

### DISCUSSION THREAD

**Discussion by murugappanrm** [nuxt/nuxt#16404] [Questions] [answered]
Hi,

Suddenly the Nuxt3 install is not working anymore. I have done a few instances a few hours ago and they all work fine. Now any new installation is not working anymore. After i execute "npm run dev" i get the "Nitro" complete state. When click the URL http://localhost:3000/ the page load up but i dont get the nuxt welcome instead i get this:

`No response returned from render handler: /__nuxt_error?url=/&statusCode=500&statusMessage&message=head.addEntry+is+not+a+function&stack=%3Cpre%3E%3Cspan+class=%22stack+internal%22%3Eat+C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/nuxt/dist/head/runtime/lib/vueuse-head.plugin.mjs:11:8%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+fn+(C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/nuxt/dist/app/nuxt.mjs:148:27)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+Object.callAsync+(/C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/unctx/dist/index.mjs:49:19)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+callWithNuxt+(C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/nuxt/dist/app/nuxt.mjs:150:23)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+applyPlugin+(C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/nuxt/dist/app/nuxt.mjs:94:29)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+Module.applyPlugins+(C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/nuxt/dist/app/nuxt.mjs:104:11)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+processTicksAndRejections+(node:internal/process/task_queues:96:5)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+async+createNuxtAppServer+(C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/nuxt/dist/app/entry.mjs:30:7)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+async+default+(/C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/@nuxt/vite-builder/dist/runtime/vite-node.mjs:27:18)%3C/span%3E%0A%3Cspan+class=%22stack+internal%22%3Eat+async+Object.renderToString+(/C:/xampp/htdocs/MyCoursesLG/nuxt3/nuxt3-bootstrap5cdn/nuxt-bscdn/node_modules/vue-bundle-renderer/dist/runtime.mjs:172:19)%3C/span%3E%3C/pre%3E&data`

Can anyone help?

**Comment by FileSoft-us** [comment]
same with nuxt RC14 edge channel broken

**Comment by murugappanrm** [comment]
This is odd. Just a minute ago, I did a fresh Nuxt3 install and it seems to work fine. I did not do anything, just did a fresh install, thats it. The only thing i did different was to wait a full 1 minute after the "Nitro built" message before i loaded the URL "http://localhost:3000/

My theory in my case is that there is some background jobs running in my PC and Nitro build ma not be have completed yet even tho the message of completion is shown. I may be wrong. I am no expert in vuejs or nuxtjs. Just a rookie.

**Reply by procrates** [reply]
The problem comes from vueuse-head.plugin.mjs since it was recently updated. Temporary solution: [https://github.com/nuxt/framework/issues/8929](https://github.com/nuxt/framework/issues/8929#issuecomment-1312551151)

**Comment by TheAlexLichter** [comment, marked as answer]
Issue was https://github.com/nuxt/framework/issues/8936, which is resolved now.
